libs/expo/betterangels/src/lib/screens/AddEditClient/PersonalInfo/index.tsx

```diff
@@ -7,6 +7,7 @@ import FullName from './FullName';
 import HmisProfiles from './HmisProfiles';
```

Comment:
Hey @mikefeldberg 

Thanks for working on this and it's almost there!
I found one thing that needs to be changed.

1. In View-Only mode, the Veteran Status should be displayed above the Living Situation field, which is the second-to-last item.

<img src="https://github.com/user-attachments/assets/bc184fed-a0e3-4fd5-9e90-af9c2d8492f7" width="320" />

Reply:
🤦 updated